src/index.tsx

```diff
@@ -90,6 +90,10 @@ const RNShare = {
       return new Promise((resolve, reject) => {
         requireAndAskPermissions(options)
           .then(() => {
+            if (options.url) {
```

Comment:
I was just about to ask if this was a breaking change since the code above now only looks for `urls` not `url`, but I see you convert it here :+1: